Lib/contextvars.py

```diff
@@ -1,4 +1,8 @@
+import collections.abc
 from _contextvars import Context, ContextVar, Token, copy_context
 
 
 __all__ = ('Context', 'ContextVar', 'Token', 'copy_context')
+
+
+collections.abc.Mapping.register(Context)
```

Comment:
For other stdlib modules in similar situations, we do this to reduce the import time of `collections.abc` (it avoids having to parse and import `collections/__init__.py`). It might be worth doing so here. I didn't make the same comment in your `multiprocessing` PR as `multiprocessing` already imports half the stdlib; I doubted it would have made a difference in that PR:

```suggestion
import _collections_abc
from _contextvars import Context, ContextVar, Token, copy_context


__all__ = ('Context', 'ContextVar', 'Token', 'copy_context')


_collections_abc.Mapping.register(Context)
```

Reply:
That makes sense